transport-native-epoll/src/test/java/io/netty/channel/epoll/EpollCompositeBufferGatheringWriteTest.java

```diff
@@ -26,7 +26,7 @@
 public class EpollCompositeBufferGatheringWriteTest extends CompositeBufferGatheringWriteTest {
     @Override
     protected List<TestsuitePermutation.BootstrapComboFactory<ServerBootstrap, Bootstrap>> newFactories() {
-        return EpollSocketTestPermutation.INSTANCE.socket();
+        return EpollSocketTestPermutation.INSTANCE.statefulSocket();
```

Comment:
Some test cases were modified to only use stateful permutations. These have all generally made sense except one, marked with a TODO below.